### ISSUE TITLE
[Fix] Make the date picker accessible using keyboard keys navigation

### ISSUE THREAD

**Issue by SukhsimranpreetSekhon** [CDH-Studio/UpSkill#1161]
**Describe the bug**
- The date picker does not work with the NVDA Screen Reader as it does not allow for travel to pick a date using the keyboard keys.
- The date picker should allow for keyboard keys to travel on its form fields.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to 'edit/create profile'
2. Click on a 'date picker field'
3. Test error

**Expected behaviour**
- The date picker should allow for the use of keyboard keys/press down to assign/pick a date to the form field. This same feature should also work in accompaniment to screen reader technologies such as NVDA.

**Screenshots**

**Additional context**
